pkg/grpc/fetch-api.proto

```diff
@@ -55,6 +55,9 @@ message SearchOptions {
   
   // exta values from the cloud event header this is a JSON string.
   google.protobuf.StringValue extras = 10;
+
+  // ID of the event to filter.
+  google.protobuf.StringValue id = 11;
```

Comment:
Two things to think about:

1. Do you want to let people pull multiple documents at once? That is, is it good for this to be a list?
2. Do we want to require people to specify source if they do choose to specify an id? My sense is that ids are just not meaningful across sources.

Reply:
Ah, I think you're following the existing pattern. I guess this is fine.